PontoFacil/PontoFacil/Views/SettingsPage.xaml

```diff
@@ -17,25 +17,28 @@
         <StackPanel>
             <TextBlock FontSize="25" Margin="0, 0, 0, 20 ">Configurações</TextBlock>
             <TextBlock>Nome do usuário</TextBlock>
-            <TextBox></TextBox>
+            <TextBox Text="{x:Bind ViewModel.Profile.Name, Mode=TwoWay}"></TextBox>
```

Comment:
Use Binding instead x:Bind to access the Profile property directly

Reply:
x:Bind have a best performance when compare with binding.